torch_geometric/graphgym/models/encoder.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, emb_dim, *args, **kwargs):
 
         self.atom_embedding_list = torch.nn.ModuleList()
 
-        for i, dim in enumerate(get_atom_feature_dims()):
+        for dim in get_atom_feature_dims():
```

Comment:
@akihironitta why do these updates affect code cov? I think it shouldn't be

Reply:
I think we can ignore it :)